internal/provider/trust_provider_resource.go

```diff
@@ -639,6 +676,8 @@ func convertTrustProviderDTOToModel(ctx context.Context, dto aembit.TrustProvide
 	switch dto.Provider {
 	case "AWSECSRole":
 		model.AwsEcsRole = convertAwsEcsRoleDTOToModel(dto)
+	case "AWSRole":
+		model.AwsRole = convertAwsEcsRoleDTOToModel(dto)
```

Comment:
Should we create a separate routine convertAwsRoleDTOToModel? Or rename convertAwsEcsRoleDTOToModel to convertAwsRoleDTOToModel?